drivers/sensor/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ add_subdirectory_ifdef(CONFIG_ISL29035		isl29035)
 add_subdirectory_ifdef(CONFIG_ISM330DHCX	ism330dhcx)
 add_subdirectory_ifdef(CONFIG_LIS2DH		lis2dh)
 add_subdirectory_ifdef(CONFIG_LIS2DS12		lis2ds12)
-add_subdirectory_ifdef(CONFIG_LIS2DW12          lis2dw12)
+add_subdirectory_ifdef(CONFIG_LIS2DW12		lis2dw12)
```

Comment:
Please move unrelated whitespace changes to a separate commit or remove from this PR

Reply:
Will fix on new PR